### PR TITLE
Additional troubleshooting for Sentry ASP .NET

### PR DESCRIPTION
**Pull request by lucas-zimerman**
This additional troubleshooting will help uses with conflicts in regard to the System.Net.Http package and Sentry.

### CODE REVIEW

File: src/platforms/dotnet/guides/aspnet/troubleshooting.mdx [hunk 1]
```diff
@@ -31,4 +31,31 @@ using System.Net;
 ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12
 ```
 
+### SentrSdk.Init with error
```
Comment (imatwawana) [inline src/platforms/dotnet/guides/aspnet/troubleshooting.mdx:34]:
```suggestion
### SentrSdk.Init With Error
```

File: src/platforms/dotnet/guides/aspnet/troubleshooting.mdx [hunk 2]
```diff
@@ -31,4 +31,31 @@ using System.Net;
...
+
+In case you receive the following error during Sentry initialization:
```
Comment (imatwawana) [inline src/platforms/dotnet/guides/aspnet/troubleshooting.mdx:36]:
```suggestion
If you get the following error during Sentry initialization, you may have a misconfiguration on your Web.config:
```

File: src/platforms/dotnet/guides/aspnet/troubleshooting.mdx [hunk 3]
```diff
@@ -31,4 +31,31 @@ using System.Net;
...
+
+`System.IO.FileNotFoundException: Could not load file or assembly System.Net.Http, Version=4.0.0.0`
+
+You may have a misconfiguration on your Web.config. You may fix it with one of the three solutions below:
```
Comment (imatwawana) [inline src/platforms/dotnet/guides/aspnet/troubleshooting.mdx:40]:
```suggestion
You can fix it with one of the three solutions below:
```

File: src/platforms/dotnet/guides/aspnet/troubleshooting.mdx [hunk 4]
```diff
@@ -31,4 +31,31 @@ using System.Net;
...
+
+* Find the `assemblyIdentity` for `System.Net.Http` and remove the `bindingRedirect`.
+```xml
+<dependentAssembly>
+  <assemblyIdentity name="System.Net.Http" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
+</dependentAssembly>
+```
+
+* Find the `assemblyIdentity` for `System.Net.Http` and lower the newVersion to `4.0.0.0`
```
Comment (imatwawana) [inline src/platforms/dotnet/guides/aspnet/troubleshooting.mdx:49]:
```suggestion
* Find the `assemblyIdentity` for `System.Net.Http` and lower the newVersion to `4.0.0.0`.
```

File: src/platforms/dotnet/guides/aspnet/troubleshooting.mdx [hunk 5]
```diff
@@ -31,4 +31,31 @@ using System.Net;
...
+```xml
+<dependentAssembly>
+  <assemblyIdentity name="System.Net.Http" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
+  <bindingRedirect oldVersion="0.0.0.0-4.2.0.0" newVersion="4.0.0.0" />
+</dependentAssembly>
+```
+
+* Install the latest package from `System.Net.Http` in NuGet into your project.
+
+More information about the problem can be found here: https://github.com/dotnet/runtime/issues/24382
```
Comment (imatwawana) [inline src/platforms/dotnet/guides/aspnet/troubleshooting.mdx:59]:
```suggestion
You can find more information about the problem [in this GitHub issue] (https://github.com/dotnet/runtime/issues/24382).
```

**Review by imatwawana — COMMENTED** [review]
Some minor wording/style changes

**Review by lucas-zimerman — COMMENTED** [review]
whoops 😂

File: src/platforms/dotnet/guides/aspnet/troubleshooting.mdx [hunk 6]
```diff
@@ -31,6 +31,33 @@ using System.Net;
...
+```xml
+<dependentAssembly>
+  <assemblyIdentity name="System.Net.Http" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
+  <bindingRedirect oldVersion="0.0.0.0-4.2.0.0" newVersion="4.0.0.0" />
+</dependentAssembly>
+```
+
+* Install the latest package from `System.Net.Http` in NuGet into your project.
+
+You can find more information about the problem [in this GitHub issue] (https://github.com/dotnet/runtime/issues/24382).
```
Comment (bruno-garcia) [inline src/platforms/dotnet/guides/aspnet/troubleshooting.mdx:59]:
```suggestion
You can find more information about the problem [in this GitHub issue](https://github.com/dotnet/runtime/issues/24382).
```

**Pull request merged**